### ISSUE TITLE
fix: Font import after webpack

### ISSUE THREAD

**Issue by richardzyx** [nexxtway/react-rainbow#1248]
-   [x] I have searched the [issues](https://github.com/nexxtway/react-rainbow/issues) of this repository and believe that this is not a duplicate.

## Describe the bug

I tried using react-rainbow components in a CRA project following Rainbow Admin setup and did a production build, encountered the error messages in console:
```
- Failed to decode downloaded font: https://team-up.discusinstitute.org/fonts/Lato-Regular/Lato-Regular.woff2
- messages:1 OTS parsing error: invalid version tag
```

The only solution is to add back the CRA default index.css to use system font and not Lato and others from Rainbow since they are imported incorrectly.

##Screenshots

The compiled script looks like this, and you can see it's set to use a relative url

 ![image](https://user-images.githubusercontent.com/7928839/71552485-03ada400-29cc-11ea-800c-03dd5123664e.png)

## To Reproduce 🕹

I created a new CRA with React Rainbow Components following the React Rainbow Admin template, deleted the CRA index.css, encountered this issue after `yarn build` & serve.

## Expected Behavior 🤔

There should be no font error or difference between my build vs. the build from React Rainbow Admin. A out-of-box CRA using Rainbow Components should be able to use the Rainbow font-family. Or Rainbow Admin can add documentation around how to get the font right without extra configuration, which I suspect is because Rainbow Admin is using an older version of Rainbow Components.

## Current Behavior 😯

Example font difference-

Font from my build:
![image](https://user-images.githubusercontent.com/7928839/71552535-d57c9400-29cc-11ea-832e-b79761770cd5.png)

React Rainbow Component:
![image](https://user-images.githubusercontent.com/7928839/71552539-e200ec80-29cc-11ea-865a-9230a63bfdc9.png)

## Context 🔦

Trying to explore React Rainbow Component and the Rainbow Admin template.

## Your Environment 🌎
**Desktop (please complete the following information):**

-   OS: macOS
-   Browser: Chrome

**Comment by richardzyx** [comment]
I should add that the same behavior & console error messages can be observed from the official website as well: https://react-rainbow.firebaseapp.com/

**Comment by TahimiLeonBravo** [comment]
@richardzyx Thanks for the feedback  
You are welcome on the react-rainbow-components community :)
 We are working on a solution to this issue.
As soon as it has been fixed we will launch a new version of react-rainbow-components.

**Comment by LeandroTorresSicilia** [comment]
Hi @richardzyx thanks for the feedback
The issue is fixed and you will be able to use our fonts in the next release.

**Comment by TahimiLeonBravo** [comment]
Hi @richardzyx 
We released a new version 1.12.0 where this issue is fixed.
Thanks for your contribution :)
